README.md

```diff
@@ -123,6 +122,24 @@ Configuration example:
 </filter>
 ```
 
+### Encoding
+
+Datadog's API expects log messages to be encoded in UTF-8.
+If some of your logs are encoded with a different encoding, we recommend using the [`record_modifier` filter plugin](https://github.com/repeatedly/fluent-plugin-record-modifier#char_encoding)
+to encode these logs to UTF-8.
+
+Configuration example:
+
+```
+# Change encoding of logs tagged with "datadog.**"
+<filter datadog.**>
```

Comment:
If this line is correct, then line 120 is probably wrong as the number of `*` is not the same.

Reply:
I'm not entirely sure about line 120, I don't have a test setup to confirm right now, I'll just go ahead and merge for now